twoliter/embedded/build.Dockerfile

```diff
@@ -138,11 +138,20 @@ RUN --mount=target=/host \
       KIT_REPOS+=("--repofrompath=${kit},/host/build/kits/${kit}/${ARCH}" --enablerepo "${kit}") ; \
     done && \
     echo "${KIT_REPOS[@]}" && \
+    declare -a EXTERNAL_KIT_REPOS && \
+    EXTERNAL_KITS=$(jq -c '.kit[]' < /host/build/external-kits/external-kit-metadata.json) && \
+    for kit in ${EXTERNAL_KITS} ; do \
+       KIT_NAME=$(jq '.vendor + "-" + .name' <<< "${kit}" | tr -d '"') && \
```

Comment:
The JSON parsing is making this too complicated. Pass the list of external kits in a space separated variable like `KIT_DEPENDENCIES` please.